www/js/ble2mqtt.js

```diff
@@ -1,137 +1,298 @@
-/* Management */
-function restart() {
-  $.ajax({
-    url: '/restart',
-    type: 'POST',
-  });
+const RESTART = '/restart';
+const OTA_UPLOAD = '/ota/';
+const BLE_DB = '/ble/bonding_db';
+const BLE_DB_WHITELIST = '/ble/bonding_db/whitelist';
+const BLE_DB_BLACKLIST = '/ble/bonding_db/blacklist';
+const CONFIG_FILE_PATH = '/fs/config.json';
+const LATEST_RELEASE = 'https://github.com/shmuelzon/esp32-ble2mqtt/releases';
+const STATUS = '/status';
+
+function progress(show = true) {
+    document.getElementById('progress').style.display = show ? 'flex' : 'none';
+    // disable\enable all buttons
+    [].concat(
+        Array.from(document.getElementsByTagName('button')), //all buttons
+        Array.from(document.getElementsByClassName('button'))  //all elements same button
+    )
+        .map(item => {
+            if (show) item.setAttribute('disabled', show)
+            else item.removeAttribute('disabled')
+        });
 }
 
-/* BLE */
-function ble_clear_bonding() {
-  $.ajax({
-    url: '/ble/bonding_db',
-    type: 'DELETE',
-  });
+function toaster(message) {
+    document.getElementById('message').style.display = 'block';
+    document.getElementById('message').innerText = message;
+    setTimeout(() => document.getElementById('message').style.display = 'none', 3000);
 }
 
-/* OTA */
-function otaStartUpload(type) {
-  var file = $('#' + type + '-file')[0].files[0];
-  if (!file)
-    return;
-
-  $('[id$=upload]').attr('disabled', true);
-  $('[id$=file]').attr('disabled', true);
-  $('#ota-spinner').show();
-  $('#ota-status').text('Uploading');
-
-  $.ajax({
-    url: '/ota/' + type,
-    type: 'POST',
-    data: file,
-    processData: false,
-  })
-  .done(onOtaUploadCompleted)
-  .fail(onOtaUploadCompleted);
+function restart() {
+    fetch(RESTART, {
+        method: 'POST',
+    })
+        .then((res) => {
+            console.log("restart")
+            window.open("/busy.html", "_self")
+        })
+        .catch((err) => {
+            document.getElementById('message').innerText = "Can't restart";
```

Comment:
Shouldn't this be a call to `toaster()`?

Reply:
yep